src/Core/View.js

```diff
@@ -278,4 +279,49 @@ View.prototype.getLayers = function getLayers(filter) {
     return result;
 };
 
+/**
+ * @typedef {object} FrameRequester
+ * @property {Function(dt, firtLoop)} update - Method that will be called each
```

Comment:
The usage in `FPContols` could be moved in `MainLoop`:
```js
frameRequester.update(this._firstLoop ? 16 : dt);
```
But let's keep it for now. What do you think of my name proposal?